src/west/_bootstrap/main.py

```diff
@@ -231,16 +221,35 @@ def bootstrap(args):
     # Clone the west source code and the manifest into west/. Git will create
     # the west/ directory if it does not exist.
 
-    clone('west repository', west_url, west_rev,
-          os.path.join(directory, WEST_DIR, WEST))
-
     clone('manifest repository', manifest_url, manifest_rev,
           os.path.join(directory, WEST_DIR, MANIFEST))
 
+    # Parse the manifest and look for a section named "west"
+    manifest_file = os.path.join(directory, WEST_DIR, MANIFEST, 'default.yml')
+    with open(manifest_file, 'r') as f:
+        data = yaml.safe_load(f.read())
+
+    if 'west' in data:
+        wdata = data['west']
+        try:
+            pykwalify.core.Core(
+                source_data=wdata,
+                schema_files=[_SCHEMA_PATH]
+            ).validate()
+        except pykwalify.errors.SchemaError as e:
+            raise e
```

Comment:
Could remove the `try` block here, since it's catching and raising the same exception.

(Can put just `raise` to re-raise an exception by the way, but not much point if there's no code between catching the exception and re-raising it.)

Reply:
Right. I will rework this to either do something useful or remove the `try` block.